pages/[username]/posts/[postId].tsx

```diff
@@ -26,10 +26,9 @@ export const PostPage = ({}: Props) => {
   const [name, setName] = useState<string>("");
   const [profilePicture, setProfilePicture] = useState<string>("");
   const [loading, setLoading] = useState(true);
+  const { username, postId } = router.query;
```

Comment:
Does moving the `router.query` outside of `fetchData()` change the behavior in any way?